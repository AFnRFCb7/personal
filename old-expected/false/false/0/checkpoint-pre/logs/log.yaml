- hash: 7257447d3b18405981da0161c0b55bcd33ccf9047bad25873c49f751adad92fe1086a373ffcc17b5913937603bc954ceee35642999d8e647edc0ee6a69abee41
  has-standard-input: "true"
  init-application: /nix/store/iw37dgjhvgqr3mfnjdra343z3mb2k2mh-init-application/bin/init-application
  links: ""
  mount: /build/resources/mounts/0000000000000002
  release-application: /nix/store/g1b7gasv2mcb134rc538yd8mwrqzbj7c-release-application/bin/release-application
  standard-error: ""
  standard-input: 91caebc6ea3ebe5b76e58d6ff22741badf8f57abf854235f20e0850d2aa310e98a8ce80eb5ed97b99c434380c6fd48a0631066cd5d3cb42ac3076de11ccf3d80
  standard-output: 0222ce319d2c8cbafe6848639aa582f0479199e8e4e4bda8e6efd915e0113d465b77c1a1a9e6984767c9267e6ebab96e4f3ffb930a83d773533985605584a1c7
  status: "0"
  transient: ""
  type: good
