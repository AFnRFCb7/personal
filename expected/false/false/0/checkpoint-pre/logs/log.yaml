- hash: d14990a4332abe0bac8a51b7f97520ffa75ecd3bb368f301ed7d7f51ed87aef82b3eb9f9a42a5a9eec1b901de082a078c4339d5c6b85e913fcd88ba74790067d
  has-standard-input: "true"
  init-application: /nix/store/iw37dgjhvgqr3mfnjdra343z3mb2k2mh-init-application/bin/init-application
  links: ""
  mount: /build/resources/mounts/0000000000000002
  release-application: "null"
  standard-error: ""
  standard-input: 91caebc6ea3ebe5b76e58d6ff22741badf8f57abf854235f20e0850d2aa310e98a8ce80eb5ed97b99c434380c6fd48a0631066cd5d3cb42ac3076de11ccf3d80
  standard-output: 0222ce319d2c8cbafe6848639aa582f0479199e8e4e4bda8e6efd915e0113d465b77c1a1a9e6984767c9267e6ebab96e4f3ffb930a83d773533985605584a1c7
  status: "0"
  transient: ""
  type: good
